pkgs/tools/networking/chisel/default.nix

```diff
@@ -17,6 +17,10 @@ buildGoModule rec {
 
   vendorSha256 = "01wh8fn76jn8hnf7gj759k8dwqrr0p36xmsxfc8dayzinpl10fqv";
 
+  postPatch = ''
+    substituteInPlace share/version.go --replace "0.0.0-src" "${version}"
```

Comment:
Please use ldflags like upstream https://github.com/jpillora/chisel/blob/master/Dockerfile#L10 does.

Reply:
Thanks, @SuperSandro2000. To late...the [docs](https://nixos.org/manual/nixpkgs/unstable/#ssec-go-common-attributes) would have shown me the exact example for the case here.